apps/mobile/src/screens/tasks/tasks.tsx

```diff
@@ -104,16 +115,50 @@ const Tasks: React.FC = () => {
 						tabs={[TaskTab.ACTIVE, TaskTab.PAST]}
 					/>
 				</View>
-				<ScrollView>
-					{tasks.map((task) => (
-						<TaskCard
-							key={task.id}
-							onComplete={handleComplete}
-							onSkip={handleSkip}
-							task={task}
-						/>
-					))}
-				</ScrollView>
+				<ImageBackground
+					resizeMode="cover"
+					source={
+						require("~/assets/images/background.png") as ImageSourcePropType
+					}
+					style={[globalStyles.flex1, styles.backgroundContainer]}
+				>
+					<View style={[globalStyles.flex1, styles.background]}>
+						{(isCurrentEmpty && mode === TasksMode.CURRENT) ||
+						(isPastEmpty && mode === TasksMode.PAST) ? (
+							<View
+								style={[
+									globalStyles.flex1,
+									globalStyles.alignItemsCenter,
+									globalStyles.justifyContentCenter,
+								]}
+							>
+								<Text
+									preset="subheading"
+									style={[globalStyles.ph32, styles.empty]}
+									weight="bold"
+								>
+									{mode === TasksMode.CURRENT
+										? "You don't have any active tasks"
+										: "You don't have any completed tasks"}
+								</Text>
+							</View>
```

Comment:
This can be moved to a separate component and used as a ListEmptyComponent prop of FlatList